### PR TITLE
Kuksa Appstore - Changed relationship of Distribution-Software

### PR DESCRIPTION
**Pull request by trademkose**
Kuksa Appstore - Changed relationship of Distribution-Software and This commit contains fixes which are related to Berlin meeting.

Signed-off-by: Adem KOSE <akose@NETAS.com.tr>

### CODE REVIEW

File: kuksa-appstore/src/main/java/org/eclipse/kuksa/appstore/client/HawkbitMultiPartFileFeignClient.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+package org.eclipse.kuksa.appstore.client;
+
+import org.eclipse.kuksa.appstore.configuration.HawkbitMultiPartFileFeignConfig;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.multipart.MultipartFile;
+
+import feign.Param;
```
Comment (rohoet) [inline kuksa-appstore/src/main/java/org/eclipse/kuksa/appstore/client/HawkbitMultiPartFileFeignClient.java]: PB CQ feign-core 9.7.0  ✅

File: kuksa-appstore/src/main/java/org/eclipse/kuksa/appstore/client/HawkbitMultiPartFileFeignClient.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * SPDX-License-Identifier: EPL-2.0
+ *  
+ * Contributors:
+ * Adem Kose, Fatih Ayvaz and Ilker Kuzu (Netas Telekomunikasyon A.S.) - Initial functionality
+ ******************************************************************************/
+package org.eclipse.kuksa.appstore.client;
+
+import org.eclipse.kuksa.appstore.configuration.HawkbitMultiPartFileFeignConfig;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PathVariable;
```
Comment (rohoet) [inline kuksa-appstore/src/main/java/org/eclipse/kuksa/appstore/client/HawkbitMultiPartFileFeignClient.java]: PB CQ org.springframework.web 4.3.9 ✅

File: kuksa-appstore/src/main/java/org/eclipse/kuksa/appstore/model/hawkbit/upload/FeignMultipartEncoder.java [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+import java.io.OutputStream;
+import java.lang.reflect.Type;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Map.Entry;
+
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpOutputMessage;
+import org.springframework.http.MediaType;
```
Comment (rohoet) [inline kuksa-appstore/src/main/java/org/eclipse/kuksa/appstore/model/hawkbit/upload/FeignMultipartEncoder.java]: PB CQ  spring-web 5.1.2 ✅

**Comment by rohoet** [comment]
Thanks for the PR!
I checked third party usage and PB'd three CQs I found.
I'd like to have another committer have a look. @bs-jokri ?

**Comment by bs-jokri** [comment]
@trademkose @rohoet does anyone of you know what the problem with this PR is. Why is it not building?

**Comment by rohoet** [comment]
It builds just fine. We have an exception though as stated in issue #29.
This PR can be merged if every third party content it uses is approved. I found the ones commented above. If there’s nothing else, we can merge it or wait until #29 is closed.

**Comment by rohoet** [comment]
@bs-jokri  If you're referring to:
> `/home/rohoe/kuksa.cloud/kuksa-appstore/src/main/java/org/eclipse/kuksa/appstore/model/User.java:[18,24] cannot find symbol`

Just remove the line `@Nullable` annotation and its import `java.annotation.Nullable`.
This has been done in the master branch already - see https://github.com/eclipse/kuksa.cloud/commit/2edb5cc4b49ef56db039d1c72a2b642e95425e60

**Pull request merged**